src/lib/mapping/encrypt.ts

```diff
@@ -1,17 +0,0 @@
-import { encrypt as encryptFunction } from "../operations/encrypt";
-import { userDetails } from "../auth/userDetails";
-
-/**
- * Encrypt data with the users JWK. This function assumes (and requires) a user is logged in and a valid string to sign.
- * @param plaintext The data in string format to sign.
- * @returns The encrypted data.
- */
-export async function encrypt(
-  plaintext: Uint8Array | string,
-): Promise<Uint8Array | string | null> {
-  const user = await userDetails();
-
-  const encryptedData = await encryptFunction(plaintext, user.sub);
-
-  return encryptedData;
```

Comment:
This could return an object with the following shape:

    {
        type: 'Buffer',
        data: [42, 104, 2, 108, ...],
    }

That `decrypt` would accept:

    const res = await othent.decrypt({
      type: 'Buffer',
      data: Array.from(encryptedData),
    });

This is no longer the case, so we should mention this as a breaking change too.